OWML.ModHelper/ModManifest.cs

```diff
@@ -36,6 +36,9 @@ public class ModManifest : IModManifest
         [JsonProperty("appIds")]
         public Dictionary<string, string> AppIds { get; private set; }
 
+        [JsonProperty("requireVR")]
+        public bool RequireVR { get; set; }//set is to be made private once transition is over
```

Comment:
Don't leave comments with the expectation of something being done about it in the future, because if it never happens then the comment just causes confusion. Make a GH issue (if it doesn't exist already) explaining the backwards-incompatible changes that will need to be made.

Reply:
it's expected to be removed in next update though

Reply:
adding issue sounds reasonable though

Reply:
I know it's expected, but it's bad practice to try trust future predictions 😅

Reply:
made a little list of RequireVR-related things to be removed later on